docs/pipelines.md

```diff
@@ -307,6 +312,30 @@ spec:
         - name: pathToContext
           value: /workspace/examples/microservices/leeroy-web
 ```
+### Specifying `Matrix` in `PipelineTasks`
+
+> :seedling: **`Matrix` is an [alpha](install.md#alpha-features) feature.**
+> The `enable-api-fields` feature flag must be set to `"alpha"` to specify `Matrix` in a `PipelineTask`.
+>
+> :warning: This feature is still in very early stage of development and is not yet functional. Do not use it.
+
+You can also provide [`Parameters`](tasks.md#specifying-parameters) through the `matrix` field:
+
+```yaml
```

Comment:
will be great to have a similar example in `finally` and [custom tasks](https://github.com/tektoncd/pipeline/blob/e471b44fe38cb60b35ad8a064d95e0d97d9ce583/docs/pipelines.md#using-custom-tasks)

Reply:
great idea, added them :)